pkg/builder/cbuildidx/builder.go

```diff
@@ -148,6 +148,12 @@ func (b CbuildIdxBuilder) build() error {
 		return nil
 	}
 
+	// no CMake orchestration needed
+	if b.Options.NoDatabase {
+		log.Info("build finished successfully!")
```

Comment:
Since there is no build involved, I believe `setup` would be more adequate:
```suggestion
		log.Info("setup finished successfully!")
```

Reply:
The option `--no-database` should be only to the `setup` command.